client/src/components/Reviews/RatingsAndReviews.jsx

```diff
@@ -236,14 +237,20 @@ class RatingsAndReviews extends React.Component {
               </div>
             </div>
             {this.state.displayedReviewsCount < compareLength &&
-              <button className="btn" onClick={() => this.handleMoreReviewsClick()}>MORE REVIEWS</button>
+              <button className="btn more-reviews-btn" onClick={(e) => {
+                this.handleMoreReviewsClick();
+                this.props.handleTrackingClick(e, e.currentTarget.className);
+              }}>MORE REVIEWS</button>
             }
-            <button className="btn" onClick={this.handleAddNewReview}>ADD A REVIEW <span className="plus-icon">+</span></button>
+            <button className="btn add-reviews-btn" onClick={(e) => {
+              this.handleAddNewReview();
+              this.props.handleTrackingClick(e, e.currentTarget.className);
```

Comment:
These are interesting, I wonder if there's a way to track it without writing the same code a bunch of times?